components/controller/constants/constants.go

```diff
@@ -36,4 +36,13 @@ const (
 	LoggingConfigVolumeName = "logging-config"
 	LoggingConfigMountPath  = "/etc/knative-kafka"
 	LoggingConfigMapName    = "knative-kafka-logging"
+
+	// Health Configuration
+	HealthConfigPort = 8082
+	HealthConfigLivenessDelay = 10
+	HealthConfigLivenessPeriod = 5
+	HealthConfigLivenessPath = "/healthz"
+	HealthConfigReadinessDelay = 10
+	HealthConfigReadinessPeriod = 5
+	HealthConfigReadinessPath = "/healthy"
```

Comment:
Do these values (delays) allow for error-free startup of the containers in k8s?  if not how were they chosen?  Any testing

Reply:
I didn't see any restarts on the channel or dispatcher pods with these settings.  I tried a Liveness delay of 5 and didn't see any problems either, but I don't have the experience or evidence to set them to anything else in particular.  I didn't try very small values (under 5 seconds).  It might be a good idea to eventually move these delays to environment variables instead of constants for experimental purposes.

From what I can tell, the liveness delay is just the amount of time before kubelet will start killing your pod for failing the test, not a delay before you can use the service -- so I'm not sure how much the specific value matters.